src/main/java/com/exasol/dbbuilder/dialects/exasol/AdapterScript.java

```diff
@@ -116,71 +74,27 @@ public Builder language(final Language language) {
             return this;
         }
 
-        /**
-         * Set the content of the adapter script.
-         * 
-         * @param content script content
-         * @return self
-         */
-        public Builder content(final String content) {
-            this.content = content;
-            return this;
-        }
-
-        /**
-         * Load the adapter script content from a file.
-         *
-         * @param path path to file containing the script content
-         * @return {@code this} for fluent programming
-         * @throws IOException in case the file could not be read
-         */
-        public Builder content(final Path path) throws IOException {
-            this.content = Files.readString(path);
-            return this;
-        }
-
-        /**
-         * Set the script content to a JAR file from BucketFS.
-         *
-         * @param scriptClass         script class to execute
-         * @param pathToJarInBucketfs path to the jar in BucketFS
-         * @return self
-         */
-        public Builder bucketFsContent(final String scriptClass, final String pathToJarInBucketfs) {
-            this.content = "%scriptclass " + scriptClass + ";\n%jar " + pathToJarInBucketfs + ";\n";
-            return this;
-        }
-
-        /**
-         * Set the name of the adapter script..
-         * 
-         * @param name name of the adapter script
-         * @return self
-         */
-        public Builder name(final String name) {
-            this.name = name;
-            return this;
-        }
-
         /**
          * Build the {@link AdapterScript}.
          * 
          * @return built {@link AdapterScript}.
          */
         public AdapterScript build() {
-            requireNotNull(this.writer, "writer");
-            requireNotNull(this.parentSchema, "parentSchema");
-            requireNotNull(this.name, "name");
+            validate();
+            final AdapterScript adapterScript = new AdapterScript(this);
+            getWriter().write(adapterScript);
```

Comment:
I guess the reason why @redcatbear didi it like that is taht the purpose of this tool lib is to create test-setups as easy as possible.
If you think we should change it, open a ticket, since it affects the whole design and is not realy part of this PR.

Reply:
The main point was compact code. It was a conscious design decision that creating a POJO is mirrored in the database. I would not do that in production code, but here it makes the tests a lot more compact.